test/compile/data/parse.test.ts

```diff
@@ -55,5 +57,29 @@ describe('compile/data/parse', () => {
       assert.isTrue(result.first instanceof BinNode);
       assert.isTrue(result.last instanceof AggregateNode);
     });
+
+    it ('should return a WindowTransform Node', () => {
+      const transform: Transform = {
+        window: [
```

Comment:
probably to add another test (or modify this test) to include other optional properties of window transform